README.md

```diff
@@ -6,6 +6,15 @@ A browser extension to improve the LibraryThing cataloguing experience. Made in
 	<img src="src/img/vbl.png" alt="vancouver black library" style="width: 100px; height: auto">
 </p>
 
+## Installation
+
+[link-chrome]: https://chrome.google.com/webstore/detail/better-librarything/hbnlneckiahefebnpdhgpohonfkkcaln 'Version published on Chrome Web Store'
+[link-firefox]: https://addons.mozilla.org/en-US/firefox/addon/betterlibrarything/ 'Version published on Mozilla Add-ons'
```

Comment:
why can't i see this lol

Reply:
needs to be published first. it's pending